build.gradle

```
@@ -0,0 +1,54 @@
+apply plugin: 'java'
+
+group = 'com.pi4j'
+version = '0.0.1'
+
+description = """Pi4J :: MINIMAL EXAMPLE :: Sample minimal project"""
+project.buildDir = 'target'
+
+tasks.withType(JavaCompile) {
```

Prefer avoiding configuring the java compilation tasks eagerly, see https://docs.gradle.org/current/userguide/task_configuration_avoidance.html
```suggestion
tasks.withType(JavaCompile).configureEach {
```